src/index.js

```diff
@@ -1,36 +1,54 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/no-mutable-exports */
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] @HunterX-7 its not good practice to disable linters for a whole file since its defeats the purpose of the linter checks, try and solve the errors produced by the linter checks